Add recommendation for 32/64-bit version (#7493)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/get-clients.md

The Microsoft Teams installation for Windows provides downloadable installers in 32-bit and 64-bit architecture.

> [!NOTE]

> > The architecture (32-bit vs. 64-bit) of Microsoft Teams is agnostic to the architecture of Windows and Office that is installed. It is recommended to prefer the 64-bit version of Microsoft Teams.

Reviewer: ```suggestion
> The architecture (32-bit vs. 64-bit) of Microsoft Teams is agnostic to the architecture of Windows and Office that is installed. We recommended the 64-bit version of Microsoft Teams.
```
